src/test/java/com/meilisearch/integration/SettingsTest.java

```diff
@@ -64,7 +64,7 @@ public void testUpdateSettingsRankingRules() throws Exception {
                     "release_date:desc",
                     "rank:desc"
                 });
-        index.waitForPendingUpdate(index.updateSettings(settings).getUpdateId());
+        index.waitForTask(index.updateSettings(settings).getUid());
```

Comment:
I think we are not using the response of this `getUid()` right?

Reply:
I think so `waitForTask` take a `task` `UID`
I can write it this way if you prefer:
```java
Task task = index.updateSettings(settings);
index.waitForTask(task.getUid());
```

Reply:
Oh I'm so sorry, I didn’t saw the closing parentheses I thought the line was: `index.waitForTask(index.updateSettings(settings).getUid();`